vuepress/docs/next/tutorials/devops/plugin-configuration.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebarDepth: 2
+---
+
+# Customize Deployment with Plugin Configuration Profiles
+
+This tutorial describes three methods to utilize configuration profiles in the OperatorConfigMap to specify resource allocation. This provides a simple way to customize the deployment parameters of Entando bundle plugins for improved efficiency.
```

Comment:
I just posted a question on ENG-3245 about how/when the profiles get applied. "Just checking, my assumption is the profile needs to be configured prior to plugin deployment, or the plugin has to be manually scaled down and up before the profile takes effect. Is that right or does the operator automatically apply the changes?" When we get an answer on that we should add a note about the behavior, probably at the top with the introduction.